src/Uno.UI/UI/Xaml/Controls/MediaPlayerElement/MediaTransportControls.MediaPlayer.cs

```diff
@@ -459,13 +459,25 @@ private void OnVolumeChanged(object sender, RangeBaseValueChangedEventArgs e)
 		{
 			if (_mediaPlayer is not null)
 			{
-				_mediaPlayer.Volume = e.NewValue;
+				_mediaPlayer.Volume = e.NewValue / 100.0;
 			}
 
 			UpdateVolumeMuteStates();
 			ResetControlsVisibilityTimer();
 		}
 
+		private void OnPlayerVolumeChanged(MediaPlayer sender, object e)
+		{
+			_ = Dispatcher.RunAsync(CoreDispatcherPriority.Normal, () =>
```

Comment:
You don't need to enqueue on the dispatcher here, you're already on the UI thread.

Reply:
Oh, that's very interesting actually, thanks for pointing that out. Let's keep the behavior as it is (everything on the UI thread) to avoid regressing exiting apps that depend on that behavior, but very good catch.